website/src/controllers/user.js

```diff
@@ -117,12 +117,26 @@ api.score = function(req, res, next) {
     }, saved.toJSON().stats));
 
     // Webhooks
+
+    // Select character data to send
+    var userData = _.pick(user.toJSON(), ['_id', '_tmp', 'stats']); // user.toJSON to copy-by-value
```

Comment:
@Lokeh I don't want to be tedious but the user object is really big :) soo, I would do something like this:

```
var userData = {
    _id: user._id,
}
```

and then `_.clone()` `stats` and maybe `_tmp` (I don't remember if it's an object or not)